make/modules/jdk.internal.le/Lib.gmk

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2015, 2025, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
+# by Oracle in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+include LibCommon.gmk
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux macosx windows), true)
+
+  $(eval $(call SetupJdkLibrary, BUILD_LIBLE, \
+      NAME := le, \
+      TOOLCHAIN := TOOLCHAIN_LINK_CXX, \
+      OPTIMIZATION := LOW, \
+      JDK_LIBS := java.base:libjava, \
```

Comment:
A quick glance through the native code revealed no obvious calls to libjava functions. Are you sure this dependency is needed?

Reply:
`CHECK_NULL` is used by the native code, originating in `jni_util.h`. But, looking at other libraries, seems that 
```
EXTRA_HEADER_DIRS := java.base:libjava,
```

should be enough (i.e. replacing `JDK_LIBS` with `EXTRA_HEADER_DIRS`). Is there some even better way?